Dockerfile.node7

```diff
@@ -0,0 +1,14 @@
+# Dockerfile for using Yarn along with Node.js 7.x
+# docker build -t yarnpkg/node-yarn:latest -f Dockerfile.node7 .
```

Comment:
Does that make it slower to build the image, as the intermediate layers aren't cached? Not that there's many layers in this Dockerfile anyways 😛